### PR TITLE
Added Files

### PR DESCRIPTION
**Pull request by MissAlexandra**
Need to add your own tenor key but 2 files in utilitys

### CODE REVIEW

File: src/modules/utility/doanaussie.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+	disabled: false,
+
+	guildOnly: false,
+	botPerms: [ "SEND_MESSAGES", "EMBED_LINKS" ],
+	userPerms: [  ],
+
+	execute: (arlo, msg, args) => {
+		var items = Array("australia","spider", "australian", "kangaroo", "Vegemite", "austrailia doesnt exist", "Aussie");
+		var item = items[Math.floor(Math.random() * items.length)];
+		console.log(item);
```
Comment (PlutonusDev) [inline src/modules/utility/doanaussie.js:24]: Bad practice to log to console.

File: src/modules/utility/doanaussie.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  "Key": "", // https://tenor.com/developer/keyregistration
+  "Filter": "off", // "off", "low", "medium", "high", not case sensitive
+  "Locale": "en_US", // Your locale here, case-sensitivity depends on input
+  "MediaFilter": "minimal", // either minimal or basic, not case sensitive
+  "DateFormat": "D/MM/YYYY - H:mm:ss A" // Change this accordingly
+});
+
+module.exports = {
+	name: "doanaussie",
+	aliases: [ "cmds", "commands" ],
```
Comment (PlutonusDev) [inline src/modules/utility/doanaussie.js:12]: Bad aliases.

File: src/modules/utility/image.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+module.exports = {
+	name: "image",
+	aliases: [ "cmds", "commands" ],
+	usage: "image image name",
+	description: "Finds random images with searched parameters",
+	disabled: false,
+
+	guildOnly: false,
+	botPerms: [ "SEND_MESSAGES", "EMBED_LINKS" ],
+	userPerms: [  ],
```
Comment (PlutonusDev) [inline src/modules/utility/image.js:20]: userPerms has been deprecated.

File: src/modules/utility/image.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
+const { RichEmbed } = require('discord.js');
```
Comment (PlutonusDev) [inline src/modules/utility/image.js:1]: RichEmbed isn't used anywhere in this file.

File: src/modules/utility/doanaussie.js [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+		Tenor.Search.Query(jamesfinal, "50").then(Results => {
+		  //console.log(Results);
+		  const randomLink = Results[Math.floor(Math.random()*(Results.length-1))];
+		  msg.channel.send(`**You Did An Aussie**\n\ ${randomLink.url}`)
+	  }).catch(console.error);
+		
+		//return message.channel.send(embed);
+		
+	  }
+			}
```
Comment (PlutonusDev) [inline src/modules/utility/doanaussie.js:43]: Fix indentation.

File: src/modules/utility/doanaussie.js [hunk 6]
```diff
@@ -0,0 +1,43 @@
+const { RichEmbed } = require('discord.js');
```
Comment (PlutonusDev) [inline src/modules/utility/doanaussie.js:1]: RichEmbed isn't used anywhere in this file.

File: src/modules/utility/doanaussie.js [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+	guildOnly: false,
+	botPerms: [ "SEND_MESSAGES", "EMBED_LINKS" ],
+	userPerms: [  ],
+
+	execute: (arlo, msg, args) => {
+		var items = Array("australia","spider", "australian", "kangaroo", "Vegemite", "austrailia doesnt exist", "Aussie");
+		var item = items[Math.floor(Math.random() * items.length)];
+		console.log(item);
+	  
+		//var items2 = Array("cute", "australian", "australia", "awful");
```
Comment (PlutonusDev) [inline src/modules/utility/doanaussie.js:26]: Remove unnecessary comments.

File: src/modules/utility/doanaussie.js [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+		//var item2 = items2[Math.floor(Math.random() * items2.length)];
+		//console.log(item2);
+		//var jamesfinal = (item+" "+item2);
+		var jamesfinal = (item);
```
Comment (PlutonusDev) [inline src/modules/utility/doanaussie.js:30]: Bad variable name. Use "selected" or similar.

**Review by PlutonusDev — CHANGES_REQUESTED** [review]
Hey, just a few changes before I can consider merging.
Please also merge into the `development` branch, **not** `main`.

Thank you!

File: src/modules/utility/doanaussie.js (hunk 8 above)
Comment (MissAlexandra) [reply src/modules/utility/doanaussie.js:30]: Sad pog

**Pull request closed without merging**